yarn.lock

```diff
@@ -133,11 +133,12 @@ color-name@1.1.3:
   integrity sha1-p9BVi9icQveV3UIyj3QIMcpTvCU=
 
 d@1:
```

Comment:
Do you want to change they yarn.lock? It should be in your update for JS deps

Reply:
Oops, that looks like an error, thanks!